src/hooks/useStakedHakka.ts

```diff
@@ -100,7 +88,7 @@ export default function useStakedHakka(): {
   );
 
   useEffect(() => {
-    throttledFetchStakedHakka(account);
+    throttledFetchStakedHakka(account as string);
```

Comment:
Do we need to check  `if (account) {...} `?

Reply:
I think, yes, we can make account validation before the function execute, which can let the function params more simple